webgallery/models.py

```diff
@@ -19,23 +19,23 @@ def __str__ (self):
       return "%s" % (self.title)
 
    
-class Slider (models.Model):
-   title = models.CharField(max_length = 200)
-   picture = models.ManyToManyField(Picture)
-   
-   def __str__ (self):
-      return "%s" % (self.title)
-   
-   def get_pictures(self):
-      return self.picture.all()
+#class ActivePic (models.Model):
+#   title = models.CharField(max_length = 200)
+#   picture = models.ManyToManyField(Picture)
```

Comment:
If it maps images to images why is it different than gallery, shouldn't it be handled when using gallery?
In my opinion it should be one way of presenting gallery controlled in the gallery setting